lib/libutils/ext/include/util.h

```diff
@@ -40,7 +40,9 @@
 #define MAX_UNSAFE(a, b)	(((a) > (b)) ? (a) : (b))
 #define MIN_UNSAFE(a, b)	(((a) < (b)) ? (a) : (b))
 
+#ifndef ARRAY_SIZE
```

Comment:
This should go in its own commit ("libutils: do not redefine ARRAY_SIZE").

Reply:
ok